File: 1-two-sum/two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        Myhash={}

        for i,n in enumerate(nums):
            diff=target-n
            if diff in Myhash:
                return (Myhash[diff],i)
            Myhash[n]=i                        
        

File: 1-two-sum/two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    
    for(let i=0;i<nums.length;i++){
        dif=target-nums[i];
        for(let j=i+1;j<nums.length;j++){
            if(nums[j]==dif){
                return [i,j];
            }
        }
    }
    
};